matsim/src/main/java/org/matsim/vehicles/VehicleUtils.java

```diff
@@ -51,17 +55,52 @@ public static VehicleType getDefaultVehicleType() {
 		return DEFAULT_VEHICLE_TYPE;
 	}
 
-    public static Id<Vehicle> getVehicleId(Person person, String mode, Config config) {
+	/**
```

Comment:
I have moved most of the logic of attaching the vehicle ids into this utility class. Do you think this is the way to go @kainagel ?

Reply:
Looking good to me.